client/wc-api/settings/operations.js

```diff
@@ -39,15 +34,11 @@ function updateSettings( resourceNames, data, fetch ) {
 	const settingsFields = [ 'woocommerce_excluded_report_order_statuses' ];
 
 	if ( resourceNames.includes( resourceName ) ) {
-		const url = NAMESPACE + '/settings/wc_admin/';
+		const url = '/wc/v3/settings/wc_admin/';
```

Comment:
We should still use `NAMESPACE` here so we don't have to find this and update it upon core merge.

That means we still need a v4 file, though it can be empty (see where we do this for product categories for example: https://github.com/woocommerce/wc-admin/blob/master/includes/api/class-wc-admin-rest-product-categories-controller.php).

These extra files will be needed for core merge anyway - See a v3 example at https://github.com/woocommerce/woocommerce/blob/master/includes/api/class-wc-rest-product-tags-controller.php

Reply:
Addressed in 8051d461.